src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/EmptyCatchBlock/context/defaultContextConfig.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="EmptyCatchBlock"/>
```

Comment:
You also should specify `commentFormat` because it will treat all comments as ignore-comment even your comments like `// violation context`, and there will be no violations.

Reply:
@baratali I move `// violation context` to another line to avoid this problem.